streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -423,10 +423,8 @@ public void process(final Record<Object, Object> record) { }
         }
     }
 
-    @Deprecated // testing old PAPI
     @Test
     public void shouldNotAllowToAddGlobalStoreWithSourceNameEqualsProcessorName() {
-        when(globalStoreBuilder.name()).thenReturn("anyName");
```

Comment:
Some require side cleanup, as we don't call `name()` again, but throw `TopologyException` earlier. So need to remove this (otherwise test fails with "unnecessary stubbing" error)